src/components/AdminSettings.vue

```diff
@@ -232,6 +242,7 @@ export default {
 		return {
 			serverMode: '',
 			serverError: Object.values(OC.getCapabilities()['richdocuments'].collabora).length > 0 ? SERVER_STATE_OK : SERVER_STATE_CONNECTION_ERROR,
+			hostErrors: [OC.getHost() === 'localhost' || OC.getHost() === '127.0.0.1', OC.getProtocol() !== 'https', false],
```

Comment:
The global OC.getHost/getProtocol functions are deprecated since they were mainly helpers to make javascript code testable in old days. `window.location.protocol` and `window.location.host` are the suggested repacements to use.

Reply:
Oh. Good to know. I had changed `window.location.protocol` and `window.location.host` to the OC calls on purpose after seeing the OC calls in the code. I'll change them back.